CONFLICTS.md

```diff
@@ -3,6 +3,10 @@
 This document explains why certain conflicts were added to `composer.json` and
 references related issues.
 
+- `symfony/password-hasher": "^6.0`:
+
+   Symfony in version 5.3 change password hashing logic, and in version 6.0 they removed BC layer
```

Comment:
I'm going to try and reword this a little bit to better explain the actual reason for conflict here.  Also, I'm not quite sure if the conflict should actually be against the Password Hasher component or the Security-Core component (or both) as Sylius is dependent on the API from the Security-Core component and not yet aware of the Password Hasher component.

```suggestion
   In Symfony 5.3, the Password Hasher component was introduced and extracted the password encoders out of the Security-Core component.  In 6.0, the B/C layer is removed.
```